### ISSUE TITLE
parallelize_loop contains std::cout

### ISSUE THREAD

**Issue by codesaurus97** [bshoshany/thread-pool#5]
Problem: The function parallelize loop of the thread_pool contains a std::cout instructions at line 106. This causes unwanted prints on the console.

```c++
    void parallelize_loop(T first_index, T last_index, const F& loop, ui32 num_tasks = 0)
    {
        if (num_tasks==0)
            num_tasks = thread_count;
        if (last_index<first_index)
            std::swap(last_index, first_index);
        size_t total_size = last_index-first_index+1;
        size_t block_size = total_size/num_tasks;
        if (block_size==0) {
            block_size = 1;
            num_tasks = std::max((ui32) 1, (ui32) total_size);
        }
        std::atomic<ui32> blocks_running = 0;
        for (ui32 t = 0; t<num_tasks; t++) {
            T start = (T) (t*block_size+first_index);
            T end = (t==num_tasks-1) ? last_index : (T) ((t+1)*block_size+first_index-1);
            std::cout << start << '-' << end << '\n';  // < -- HERE IT IS :)
            blocks_running++;
            push_task([&start, &end, &loop, &blocks_running] {
                for (T i = start; i<=end; i++)
                    loop(i);
                blocks_running--;
            });
            while (blocks_running!=0) {
                std::this_thread::yield();
            }
        }
    }
```

**Comment by bshoshany** [comment]
Oh damn! That was something I added while debugging and apparently forgot to remove it. Thanks, it will be removed in the next version :) Please let me know if you find any other problems like that.

**Comment by bshoshany** [comment]
Closing since this issue is now fixed in v1.3.
